app/panel/components/Summary.jsx

```diff
@@ -20,15 +20,17 @@ import { DynamicUIPortContext } from '../contexts/DynamicUIPortContext';
 import { sendMessage } from '../utils/msg';
 import globals from '../../../src/classes/Globals';
 import {
-	CliqzFeatures,
```

Comment:
On second glance this file still looks fine.  I will look a third time though as it is big.

Reply:
Cool beans ~